src/components/selects-request/archive-list/index.tsx

```diff
@@ -0,0 +1,44 @@
+import styled from 'styled-components';
+
+import { useState } from 'react';
+
+import { useSelectsRequestList } from '@src/hooks/api/Super/SelectsRequest';
+import ArchiveTable from './table';
+
+import { colums } from './table/colums';
+
+function ArchiveList() {
+  const { data: items, mutate, response } = useSelectsRequestList({
+    isArchived: true,
+  });
+
+  return (
+    <Wrapper>
+      <ArchiveTable
+        dataSource={items}
+        colums={colums}
+        dataMapper={(data) => {
+          return {
+            ...data,
+            userName: data.user.name,
+            userUploaderType: data.user.uploaderType,
+            productBrand: data.item.brandName,
+            product: data.item.name,
+            option: data.product.options.join('/'),
+            assignee: data.assignee.name,
+          };
+        }}
+      />
+    </Wrapper>
```

Comment:
```suggestion
  const mapTableData = (data) => {
    return {
      ...data,
      userName: data.user.name,
      userUploaderType: data.user.uploaderType,
      itemBrandName: data.item.brandName,
      itemName: data.item.name,
      option: data.product.options.join('/'),
      assigneeName: data.assignee.name,
    };
  }

  return (
    <Wrapper>
      <ArchiveTable
        dataSource={items}
        columns={columns}
        dataMapper={mapTableData}
      />
    </Wrapper>
```

1. 인라인 함수 사용을 지양해주세요!
2. colums -> columns 오타 수정했습니다.
3. 변수 네이밍의 통일성을 유지하면 좋을 것 같아요.
  user.name -> userName
  user.uploaderType -> userUploaderType 이므로
  item.brandName -> itemBrandName
  item.name -> itemName
  assignee.name -> assigneeName 으로 변경하면 좋을 것 같습니다!
4. data parameter를 타이핑 해보세요!

Reply:
538b767